tests/middleware/test_message_logger.py

```diff
@@ -37,8 +37,8 @@ async def app(scope, receive, send):
     with caplog_for_logger(caplog, "uvicorn.asgi"):
         caplog.set_level(TRACE_LOG_LEVEL, logger="uvicorn.asgi")
         caplog.set_level(TRACE_LOG_LEVEL)
-        app = MessageLoggerMiddleware(app)
-        async with httpx.AsyncClient(app=app, base_url="http://testserver") as client:
+        transport = httpx.ASGITransport(MessageLoggerMiddleware(app))
+        async with httpx.AsyncClient(transport=transport, base_url="http://testserver") as client:
```

Comment:
Can we apply this style on the other tests, please? i.e. `transport` in a single line, and then `transport=transport`.